.swiftlint.yml

```diff
@@ -9,6 +9,7 @@ disabled_rules:
   - file_length
   - empty_count
   - type_name
+  - shorthand_operator
```

Comment:
Thank you.

Reply:
This should be there. Swift lint sometimes is very annoying.